pkg/providers/add-label/event.go

```diff
@@ -38,34 +39,72 @@ func (l *Label) processComment(event *github.IssueCommentEvent, comment string)
 	return nil
 }
 
+func (l *Label) checkLabels(labels []string) (legalLabels, illegalLabels []string, err error) {
+	repoLabels, _, e := l.provider.ListLabelsOnGithub()
+	legalLabels = []string{}
+	illegalLabels = []string{}
+	if e != nil {
+		err = errors.Wrap(e, "list labels failed")
+		return
+	}
+	repoLabel := map[string]bool{}
+	for _, l := range repoLabels {
+		repoLabel[strings.ToLower(*l.Name)] = true
+	}
+
+	for _, label := range labels {
+		if repoLabel[strings.ToLower(label)] {
+			legalLabels = append(legalLabels, label)
+		} else {
+			illegalLabels = append(illegalLabels, label)
+		}
+	}
+	return legalLabels, illegalLabels, nil
+}
+
 func (l *Label) processLabel(event *github.IssueCommentEvent, raw string) error {
-	if !l.opr.Member.IfMember(event.GetSender().GetLogin()) {
+	if !l.provider.IfMember(event.GetSender().GetLogin()) {
 		return nil
 	}
+	issueID := event.GetIssue().GetNumber()
 
 	var labels []string
 	for _, label := range strings.Split(raw, ",") {
 		labels = append(labels, strings.TrimSpace(label))
 	}
-	_, _, err := l.opr.Github.Issues.AddLabelsToIssue(context.Background(), l.owner, l.repo,
-		event.GetIssue().GetNumber(), labels)
+	legal, illegal, err := l.checkLabels(labels)
+	if err != nil {
+		return err
+	}
+	if len(legal) > 0 {
+		_, _, err = l.provider.GithubClient().Issues.AddLabelsToIssue(context.Background(), l.owner, l.repo, issueID, labels)
+	}
 
-	return errors.Wrap(err, "add labels")
+	if len(illegal) > 0 {
+		comment := fmt.Sprintf("These  labels are not found %s.", strings.Join(illegal, ","))
+		util.Println("errMsg", comment)
+		err = l.provider.CommentOnGithub(issueID, comment)
+	}
+	if err != nil {
+		return errors.Wrap(err, "add labels")
+	}
+	return nil
```

Comment:
`return errors.Wrap(err, "add labels")` returns nil if err is nil, do not need to determine if it is nil.